back-end/index.js

```diff
@@ -1,32 +1,46 @@
 import app from "./server.js"
-import mongodb from "mongodb"
+// import mongodb from "mongodb"
+import mongoose from "mongoose"
```

Comment:
is there a specific reason we switched to mongoose instead of the native mongodb package?

Reply:
I realized that it would be easier to work with mongoose as the schemas were built using Mongoose, so it would be easier to connect two things if we also used it in our MongoDB config file as well.

Reply:
Got it that was a good call!